client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -357,6 +380,15 @@ export const Admin = () => {
             <p>Professor data import to Classes is complete!</p>
           </div>
 
+          <div hidden={!(updatingSubjects === 1)} className="">
```

Comment:
This is the way that we've left warnings for updating the database on the admin page in the past, but I wonder if there's a better visual display / loading bar we can show for progress on the update.